ebpf/tls-uprobe-trace/ca-cert.pem

```diff
@@ -0,0 +1,93 @@
+Certificate:
```

Comment:
Do we already have these in wolfssl-examples/certs/? If not please document where these came from so we can make sure they are updated. I'd prefer if possible to use the example keys/certs in the root certs dir. Thanks.